docs/sphinx/index.rst

```diff
@@ -135,7 +135,7 @@ If the target Spack package supports Spack's testing hooks, you can run tests du
 For details on Spack's spec syntax, see the `Spack Specs & dependencies <http://spack.readthedocs.io/en/latest/basic_usage.html#specs-dependencies>`_ documentation.
 
 
-Uberenv looks for configuration yaml files under ``scripts/uberenv/spack_config/{platform}`` or you can use the **--spack-config-dir** option to specify a directory with compiler and packages yaml files to use with Spack. See the `Spack Compiler Configuration <http://spack.readthedocs.io/en/latest/getting_started.html#manual-compiler-configuration>`_
+Uberenv looks for configuration yaml files under ``scripts/uberenv/spack_configs/{platform}`` or under ``.uberenv/spack_configs/{platform}``. You may also use the **--spack-config-dir** option to specify a directory with compiler and packages yaml files to use with Spack. See the `Spack Compiler Configuration <http://spack.readthedocs.io/en/latest/getting_started.html#manual-compiler-configuration>`_
```

Comment:
Is this a typo fix? Or a change from `scripts/uberenv/spack_config` to `scripts/uberenv/spack_configs` ?

Reply:
Typo fix.